From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Galt <johngaltfirstrun@gmail.com>
Date: Fri, 28 Mar 2025 16:04:33 -0400
Subject: [PATCH 25/32] libc: Set 100 instr-limit

Do not additionally restrict instr limit for libc.

Benefits app open performance.

[MikolajQ: Ported to Android 17: libc now takes afdo from libc_ndk_defaults, the property stays on libc itself]

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 libc/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libc/Android.bp b/libc/Android.bp
index 73cda9a..a5e69ba 100644
--- a/libc/Android.bp
+++ b/libc/Android.bp
@@ -1727,6 +1727,7 @@ cc_library {
     defaults: [
         "libc_ndk_defaults",
     ],
+    lto_Instr100: true,
 }
 
 cc_library_static {
